.github/workflows/do-not-merge.yml

```diff
@@ -1,4 +1,4 @@
-name: Unmergable Labels Check
+name: Unmergaeble Labels Check
```

Comment:
I think you have a typo in your typo fix

Reply:
**unmergeable** is now officially a tricky word, thank you for spotting that one 😅